pkgs/tools/admin/afterburn/default.nix

```
@@ -0,0 +1,28 @@
+{ stdenv, openssl, pkg-config, fetchCrate, rustPlatform }:
+
+with rustPlatform;
+
+buildRustPackage rec {
```

since this is the only thing that exists on rustPlaform, i would just make it explicit here and remove the `with rustP..;` on line 3
```suggestion
rustPlatform.buildRustPackage rec {
```